internal/endtoend/testdata/func_args/go/query.sql.go

```diff
@@ -33,9 +33,9 @@ const makeIntervalSecs = `-- name: MakeIntervalSecs :one
 SELECT make_interval(secs => $1)
 `
 
-func (q *Queries) MakeIntervalSecs(ctx context.Context, secs interface{}) (interface{}, error) {
+func (q *Queries) MakeIntervalSecs(ctx context.Context, secs float64) (int64, error) {
```

Comment:
The compiler is now smart enough to pull out type information for named parameters in function calls.